### ISSUE TITLE
instant search around lat lng to typesense geo search

### ISSUE THREAD

**Issue by weswelker88** [typesense/typesense-instantsearch-adapter#63]
## Description

I am currently attempting to use the ```<ais-config>```  attribute ```aroundLatLng``` to perform a search based on an the radius around an object's lattitude and longitude. 

## Steps to reproduce

My ```ais-config``` tag looks like:
```
<ais-configure
        :aroundLatLng="aroundLatLng"
        :hits-per-page.camel="10"
/>
```
Where the prop ```aroundLatLng``` is 
```
'_geoloc:(48.90615915923891, 2.3435897727061175, 5.1 km)'
```

## Expected Behavior

I was expecting to see the results within a 5.1 km radius from the lat and lng entered

## Actual Behavior

I got:
![image](https://user-images.githubusercontent.com/18431649/127916815-9347d0d9-1db7-4cf6-83ca-772c0c8c0f5c.png)

## Metadata

**Typsense Version**: 0.21.0

**OS**: macOS ver 11.4

**Comment by jasonbosco** [comment]
@weswelker88 I just added support for `aroundLatLng` and `aroundRadius` in `v2.1.0-0` of the adapter. Could you upgrade the adapter and try this again?

On a side note, may I know what your use-case is for adding `aroundLatLng` without `aroundRadius`? Are you using it just to geo sort the results?

**Comment by weswelker88** [comment]
I am using it to filter results based based on a lat lng

**Comment by jasonbosco** [comment]
I see, are you also using the geosearch widget in instantsearch? 

In any case, does `v2.1.0-0` solve the issue you saw?

**Comment by jasonbosco** [comment]
I've published v2.1.0 publicly and confirmed that it works via tests. But please feel free to reopen this if it still doesn't work.
